cmd/gossamer/main_test.go

```diff
@@ -193,7 +227,56 @@ func TestInvalidCommand(t *testing.T) {
 	}
 }
 
-// TODO: TestGossmaerCommand test "gossamer" does not error
+func TestGossamerCommand(t *testing.T) {
+	basePort := 7000
+	currentDir, err := os.Getwd()
+	require.Nil(t, err)
+
+	genesisPath := filepath.Join(currentDir, "../..", "node/gssmr/genesis.json")
```

Comment:
You can use `utils.GetGssmrGenesisPath()` here within the `lib/utils` package.

Reply:
reminder of this